config/filament.php

```diff
@@ -262,7 +262,7 @@
     |
     */
 
-    'favicon' => public_path('img/speedtest-tracker-icon.png'),
+    'favicon' => '/img/speedtest-tracker-icon.png',
```

Comment:
Good catch but the correct way to reference the file in the Laravel framework would be as an asset, can you update your pr to reference `asset()` helper method instead of `public_path()` and I'll merge it through?

Reply:
Yes, sure